include/fastdds/dds/core/policy/QosPolicies.hpp

```diff
@@ -1300,19 +1676,18 @@ class ResourceLimitsQosPolicy : public Parameter_t, public QosPolicy
 
 
 /**
- * Class DurabilityServiceQosPolicy, to indicate the Durability Service.
- * This QosPolicy can be defined and is transmitted to the rest of the network but is not implemented in this version.
- * service_cleanup_delay: Default value c_TimeZero.
- * history_kind: Default value KEEP_LAST_HISTORY_QOS.
- * history_depth: Default value 1.
- * max_samples: Default value -1.
- * max_instances: Default value -1.
- * max_samples_per_instance: Default value -1.
+ * Specifies the configuration of the durability service. That is, the service that implements the DurabilityQosPolicy kind
+ * of TRANSIENT and PERSISTENT.
```

Comment:
```suggestion
 * of PERSISTENT.
```

Reply:
In the DDS Standard states that this QoS Policy is applied if the Durability is either TRANSIENT or PERSISTENT.